nixos/modules/services/web-apps/writefreely.nix

```diff
@@ -89,9 +107,14 @@ let
       optionalString (cfg.database.passwordFile != null)
       "$(head -n1 ${cfg.database.passwordFile})"
     }
+    oauth_client_secret=${
```

Comment:
It may make more sense to only include the logic for writing the parity secret if the user has enabled the feature. We probably don't need to have this and the associated sed invocation in the script unless there is a secret to write.